aspnetcore/security/authentication/windowsauth.md

```diff
@@ -87,11 +92,11 @@ Using Visual Studio or the dotnet CLI, *publish* your app to the destination fol
 
 Learn more about [publishing to IIS](https://docs.microsoft.com/en-us/aspnet/core/publishing/iis).
 
-After completing these steps, you should be able to launch your app from its IIS-bound path and Windows authentication should work correctly.
+Launch the app to verify Windows authentication is working.
 
 ## Enabling Windows authentication with WebListener
 
-Although Kestrel doesn't support Windows authentication, you can use [WebListener](/fundamentals/servers/weblistener) to support self-hosted scenarios on Windows. Use the following code snippet to configure your app's web host to use WebListener with Windows authentication:
+Although Kestrel doesn't support Windows authentication, you can use [WebListener](/fundamentals/servers/weblistener) <!-- BAD - xref only!!!! --> to support self-hosted scenarios on Windows. The following example configures the app's web host to use WebListener with Windows authentication:
```

Comment:
What is xref? Searched CONTRIBUTING.md and didn't find any reference to it. What should this link be?